block.go

```diff
@@ -35,36 +35,18 @@ func (bc *BlockClient) GetChildren(ctx context.Context, id BlockID, pagination *
 		return nil, err
 	}
 
-	var response struct {
-		Object     ObjectType `json:"object"`
-		NextCursor string     `json:"next_cursor"`
-		HasMore    bool       `json:"has_more"`
-		Results    []map[string]interface{}
-	}
-	err = json.NewDecoder(res.Body).Decode(&response)
+	response := &GetChildrenResponse{}
+	err = json.NewDecoder(res.Body).Decode(response)
 	if err != nil {
```

Comment:
Kept it like this because it seemed more idiomatic. Could technically become
```go
response := &GetChildrenResponse{}
err = json.NewDecoder(res.Body).Decode(response)
return response, err
```
but again, seems more idiomatic to return a nil pointer.